src/zc/table/table.py

```diff
@@ -382,12 +389,14 @@ def _header_template(self, options):
             <span class="zc-table-sortable"
                   onclick="javascript: %(script_name)s(
                         '%(columnName)s', '%(sort_on_name)s')"
-                    onMouseOver="javascript: this.className='sortable zc-table-sortable'"
-                    onMouseOut="javascript: this.className='zc-table-sortable'">
+                  onMouseOver="javascript:
```

Comment:
Is it good to split an attribute like this?

Reply:
It should be fine.

Reply:
+1